app/assets/javascripts/application.js

```diff
@@ -133,7 +131,6 @@ var initialize_modules = function() {
   App.SendAdminNotificationAlert.initialize();
   App.Managers.initialize();
   App.Globalize.initialize();
-  App.SendAdminNotificationAlert.initialize();
```

Comment:
@aitbw So sorry for not noticing earlier, and for adding more comments on this PR. I've just realized the lines deleted in the main CONSUL repo are actually the other ones :sweat_smile:, as can be seen in [application.js in CONSUL](https://github.com/consul/consul/blob/master/app/assets/javascripts/application.js#L47).

Just wondering if we could change this PR and delete the other 3 lines instead of these ones, so we've got both files in sync and avoid conflicts in the future. Sorry for the inconvenience!

Reply:
Could you clarify your comment, @javierm ? I'm quite confused on what is it that you need :sweat_smile:

Reply:
Hahaha, sorry I didn't explain it properly, I know it's confusing :smile:. What I'm suggesting is, instead of deleting lines 51, 88, and 136 (as done here), delete lines 49, 85 and 133. Because, if we merge this PR as it is right now, this file would have contents like:

```javascript
App.SendAdminNotificationAlert.initialize();
App.Managers.initialize();
App.Globalize.initialize();
```
While the file in the CONSUL repo has contents like:
```javascript
App.Managers.initialize();
App.Globalize.initialize();
App.SendAdminNotificationAlert.initialize();
```
And this small difference might lead to conflicts in the future.